clients/web/src/routes/auth/org.svelte

```diff
@@ -0,0 +1,61 @@
+<script lang="ts" context="module">
+    import type {Load} from "@sveltejs/kit";
+    import cookies from "js-cookie";
+
+    export const load: Load = ({session}) => {
+        if (!session.user) {
+            return {redirect: "/auth/login", status: 302};
+        }
+        return {};
+    };
+</script>
+
+<script lang="ts">
+    import {goto} from "$app/navigation";
+    import {session} from "$app/stores";
+    import {currentUser} from "$lib/api";
+    import {Card, CenteredCardLayout} from "$lib/components";
+
+    import {switchOrganizationMutation} from "../../lib/api/mutations/SwitchOrganization.mutation";
+    import Spinner from "../../lib/components/atoms/Spinner.svelte";
+    import {type SessionUser, constants, extractJwt} from "../../lib/utils";
+
+    async function login(orgId: number): Promise<void> {
+        const {
+            switchOrganization: {access, refresh},
+        } = await switchOrganizationMutation({organizationId: orgId});
+
+        $session.user = extractJwt<SessionUser>(access);
+        $session.token = access;
+
+        cookies.set(constants.auth_cookie_key, access, {expires: 0.25}); // 6 hours
+        cookies.set(constants.refresh_token_cookie_key, refresh, {
+            expires: 7,
+        });
```

Comment:
Where is this logic done for login elsewhere? We don't want to have 2 different ways of setting these cookies